static/app/views/issueDetails/streamline/eventDetailsHeader.tsx

```diff
@@ -297,18 +309,20 @@ const DateFilter = styled(TimeRangeSelector)`
 
 const GraphSection = styled('div')`
   display: flex;
-  &:not(:first-child) {
-    border-top: 1px solid ${p => p.theme.translucentBorder};
+  gap: ${p => p.theme.space.md};
+  & > div {
```

Comment:
This selector is pretty broad and may mess with someone editing the graph later on, is there a better way to target the specific component we want to style?